oe_corporate_blocks.module

```diff
@@ -33,3 +41,28 @@ function oe_corporate_blocks_theme() {
 function oe_corporate_blocks_locale_translation_projects_alter(&$projects) {
   $projects['oe_corporate_blocks']['info']['interface translation server pattern'] = drupal_get_path('module', 'oe_corporate_blocks') . '/translations/%project-%language.po';
 }
+
+/**
+ * Set site owner on given render array, if any.
+ *
+ * This is set as preprocess function on oe_corporate_blocks_theme().
+ *
+ * @param array $variables
+ *   Render array variables.
+ */
+function oe_corporate_blocks_preprocess_set_site_owner(array &$variables): void {
+  $cacheability = CacheableMetadata::createFromRenderArray($variables);
+  $configuration = \Drupal::configFactory()->get('oe_corporate_site_info.settings');
```

Comment:
Mh, not sure it about it, I'd probably use the service to the the configuration name, but not much else, since this is a rendering use case, I'm not sure the service should care about this. I'll use `SiteInformation:CONFIG_NAME` here instead.